lib/core/imports/index.js

```diff
@@ -5,30 +5,18 @@ import doT from '@deque/dot';
 import emojiRegexText from 'emoji-regex';
 import memoize from 'memoizee';
 
-import Promise from 'core-js-pure/features/promise';
-import Uint32Array from 'core-js-pure/features/typed-array/uint32-array';
-import WeakMap from 'core-js-pure/es/weak-map';
+import es6promise from 'es6-promise';
+import { Uint32Array } from 'typedarray';
+import 'weakmap-polyfill';
 
-/**
- * Polyfill `WeakMap`
- * Reference https://github.com/zloirock/core-js/
- */
 if (!('WeakMap' in window)) {
 	window.WeakMap = WeakMap;
 }
 
```

Comment:
I believe this block is no longer need. It looks like weakmap-polyfill checks if `self.WeakMap` exists before doing anything and then sets it on the global `self` itself. https://github.com/polygonplanet/weakmap-polyfill/blob/master/weakmap-polyfill.js#L11-L28

```suggestion
```